pyopia/cli.py

```diff
@@ -239,6 +224,24 @@ def merge_mfdata(path_to_data: str, prefix='*', overwrite_existing_partials: boo
                                        chunk_size=chunk_size)
 
 
+def process_file_list(file_list, pipeline_config, c):
```

Comment:
Can we have a docstring to explain what this function does, and what the input arguments are?

Reply:
Sure, I will add a docstring asap.